# Small hacks for cairo fast image rendering
# Disabled FILTER image
#
Index: mozilla/gfx/cairo/cairo/src/cairo-xlib-surface.c
===================================================================
--- mozilla.orig/gfx/cairo/cairo/src/cairo-xlib-surface.c
+++ mozilla/gfx/cairo/cairo/src/cairo-xlib-surface.c
@@ -957,16 +957,18 @@
     if (!CAIRO_SURFACE_RENDER_HAS_FILTERS (surface))
     {
 	if (filter == CAIRO_FILTER_FAST || filter == CAIRO_FILTER_NEAREST)
 	    return CAIRO_STATUS_SUCCESS;
 
 	return CAIRO_INT_STATUS_UNSUPPORTED;
     }
 
+	filter = CAIRO_FILTER_FAST;
+
     switch (filter) {
     case CAIRO_FILTER_FAST:
 	render_filter = FilterFast;
 	break;
     case CAIRO_FILTER_GOOD:
 	render_filter = FilterGood;
 	break;
     case CAIRO_FILTER_BEST:
